config/initializers/resque_config.rb

```diff
@@ -3,4 +3,4 @@
 Resque.redis = Redis.new(host: config[:host], port: config[:port], thread_safe: true)
 
 Resque.inline = Rails.env.test?
-Resque.redis.namespace = "heliotrope:#{Rails.env}"
+Resque.redis.namespace = "#{Settings.resque_namespace}:#{Rails.env}"
```

Comment:
Good point. Especially once I differentiated dev and test which were both just "heliotrope" previously.